tools/elf2hunk.c

```diff
@@ -872,7 +924,7 @@ int elf2hunk(int file, int hunk_fd, const char *libname, int flags)
     	if (hh[i]==NULL || hh[i]->hunk < 0)
     	    continue;
 
-    	count = (hh[i]->size + 4) / 4;
+        count = (hh[i]->size + 3) / 4;
```

Comment:
Just by looking at this code I cannot figure out why you changed from 4 to 3. Could you replace magic number with some symbolic value? Does `/ 4` really means `/ sizeof(ULONG)` ?